core/src/components/toast/toast.tsx

```diff
@@ -484,6 +496,21 @@ export class Toast implements ComponentInterface, OverlayInterface {
     return buttons;
   }
 
+  private getShape(): string | undefined {
+    const { theme, shape } = this;
+
+    // TODO(ROU-11300): Remove theme check when shapes are defined for all themes.
+    if (theme !== 'ionic') {
+      return undefined;
+    }
+
+    if (theme === 'ionic' && shape === undefined) {
```

Comment:
```suggestion
    if (shape === undefined) {
```
There's no need to do this extra check when the first if statement verifies that only `ionic` would be the theme at this point.